packages/components/src/menu-group/README.md

```diff
@@ -1,6 +1,31 @@
 # MenuGroup
 
-## Usage
+`MenuGroup` wraps a series of related `MenuItem` components into a common section.
+
+![MenuGroup Example](https://wordpress.org/gutenberg/files/2019/03/MenuGroup.png)
+
+1. MenuGroup
+
+## Table of Contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+## Design guidelines
+
+### Usage
+
+A `MenuGroup` should be used to indicate that two or more individual MenuItems are related. When other menu items exist above or below a `MenuGroup`, the group should have a divider line between it and the adjacent item. A MenuGroup can optionally include a label to describe its contents. 
```

Comment:
No tick marks around MenuGroup for some reason

```suggestion
A `MenuGroup` should be used to indicate that two or more individual MenuItems are related. When other menu items exist above or below a `MenuGroup`, the group should have a divider line between it and the adjacent item. A `MenuGroup` can optionally include a label to describe its contents. 
```
Should "MenuItems" have tick marks?

Reply:
Yep, it should. Good catch! I've opened #14490 as a quick fix for that. 🙂